lib/improver/spotdata/extract_data.py

```diff
@@ -318,7 +318,8 @@ def make_cube(self, cube, data, sites):
         # associated with dimensions cannot be preserved as the dimensions will
         # be reshaped and the auxiliarys no longer compatible.
         scalar_coordinates = [coord.name() for coord in
-                              cube.coords(dimensions=[])]
+                              cube.coords(dimensions=[])
+                              if coord.name() != 'forecast_period']
```

Comment:
Please can you add something to the comment about why forecast period is now excluded (i.e. forecast_period worked out below)